CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ All notable changes to this project will be documented in this file.
 
 #### Fixed
 
+## 4.2.1
```

Comment:
Hm, this one doesn't really need a new release as all of the stuff that changed is for the infrastructure rather than the project itself.